content/templates/content/tip.html

```diff
@@ -31,25 +31,31 @@
             </div>
         </div>
 
-        <div>
-        {% if page.body %}
-        <div id="tip-nav">
-            <button onclick="mySwipe.prev()">PREV</button>
-            <ul>
-            {% for block in page.body %}
-                {% if block.block_type == 'paragraph' %}
-                    <li>&#8226</li>
-                {% endif %}
-            {% endfor %}
-            </ul>
-            <button onclick="mySwipe.next()">NEXT</button>
-        </div>
-        {% endif %}
-        </div>
+        <div id="space">
+            {% if page.body %}
+            <div id="footer">
+                <div id="tip-nav">
+                    <button onclick="mySwipe.prev();empDot(-1);">PREV</button>
+                    <ul>
+                        {% for block in page.body %}
+                            {% if block.block_type == 'paragraph' %}
+                               <li>&#8226</li>
```

Comment:
Careful with the character code. Some devices don't like it when you forget to close a character code with a semicolon (`;`).